resources/lib/telstra_auth.py

```diff
@@ -0,0 +1,86 @@
+#-------------------------------------------------------------------------------
+import requests
+import collections
+import json
+import urlparse
+import config
+import comm
+import utils
+from exception import AFLVideoException
+
+class SortedHTTPAdapter(requests.adapters.HTTPAdapter):
+    def add_headers(self, request, **kwargs):
+        if 'header_order' in globals():
```

Comment:
Can we remove the globals? Could we just have a header_order variable at the top of this file maybe?